R/mcmc.R

```diff
@@ -114,10 +114,34 @@ fit_mcmc <- function(
 
 
 
-#' Title - TODO
+#' Transform array into list of arrays
+#'
+#' @description
+#' Transform an array into list of arrays where the listing
+#' is performed on a given dimension.
+#'
+#' @param a Array with number of dimensions at least 2.
+#' @param n Positive integer. Dimension of `a` to be listed.
+#'
+#' @details
+#' For example, if `a` is a 3 dimensional array and `n = 1`,
+#' `split_dim(a,n)` returns a list of 2 dimensional arrays (i.e.
+#' a list of matrices) where each element of the list is `a[i, , ]`, where
```

Comment:
Its hard to understand what this function is doing from the worded description. Perhaps just provide a small example.  For instance I did something similar in `invert_indexes()`

Reply:
ok, done